internal/controller/imagerepository_controller.go

```diff
@@ -53,6 +53,10 @@ const (
 	additionalUsersConfigMapName         = "image-controller-additional-users"
 	additionalUsersConfigMapKey          = "quay.io"
 	skipRepositoryDeletionAnnotationName = "image-controller.appstudio.redhat.com/skip-repository-deletion"
+	requestRequeueSecondsFirst           = 5
+	requestRequeueSecondsSecond          = 60
+	waitSecondsForComponentFirst         = 2 * 60
```

Comment:
What about:
`waitForRelatedComponentActiveWindowDuration`
`waitForRelatedComponentWindowDuration` (maybe fallback widow or something)
and then the corresponding delays:
`waitForRelatedComponentActiveWindowDelay`
`waitForRelatedComponentWindowDelay`

Reply:
will use

	waitForRelatedComponentInitialDelay
	waitForRelatedComponentFallbackDelay
	waitForRelatedComponentInitialWindowDuration
	waitForRelatedComponentFallbackWindowDuration